test/algorithms/buffer/buffer_linestring_geo.cpp

```diff
@@ -85,60 +109,60 @@ void test_linestring()
     std::set<int> const curved_cases_min_area{86, 181};
     // Cases which are curved such that the max area is larger than expected.
     std::set<int> const curved_cases_max_area{5, 95, 119, 142};
-    // Cases which are rounded such that it results in a large area
-    std::set<int> const round_cases_max_area{};
-    // Cases which are not yet valid or false negatives
-    std::set<int> const round_cases_invalid{};
 
-    for (auto i = 0; i < n; i++)
+#if defined(BOOST_GEOMETRY_TEST_FAILURES)
+    std::set<int> const skip_cases_round_round{};
+    std::set<int> const skip_cases_round_flat{};
+    std::set<int> const skip_cases_miter_flat{};
+#else
+    // Cases where the algorithm is still failing.
+    std::set<int> const skip_cases_round_round{17, 22, 38, 181, 196};
+    std::set<int> const skip_cases_round_flat{17, 22, 38, 103, 196};
+    std::set<int> const skip_cases_miter_flat{17, 18, 22, 38, 103, 196};
```

Comment:
196 was already skipped (in another way, see below).
Two other cases were already skipped but are now fine.